packages/phrases/src/locales/zh-cn.ts

```diff
@@ -10,6 +10,8 @@ const translation = {
     done: '完成',
     search: '搜索',
     save_changes: '保存更改',
+    loading: '读取中...',
+    redirecting: '页面跳转中...',
```

Comment:
nit: better replace `...` with `……`?

Reply:
Discussed offline and we are going to keep this as this English period visually looks better